dbms/src/Flash/Pipeline/GetResultSink.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 PingCAP, Ltd.
```

Comment:
This file and the containing data structure don't seem to belong to this directory. Maybe we should move them to `Operators` and `plans` directories seperately.

Reply:
ok, updated.